benefits/core/migrations/0002_sample_data.py

```diff
@@ -108,6 +108,8 @@ def load_sample_data(app, *args, **kwargs):
         unverified_title=_("eligibility.pages.unverified.mst_login.title"),
         unverified_headline=_("eligibility.pages.unverified.mst_login.headline"),
         unverified_blurb=_("eligibility.pages.unverified.mst_login.p[0]"),
+        enrollment_start_header=_("enrollment.pages.index.mst_login.start.header"),
+        enrollment_start_message=_("enrollment.pages.index.mst_login.start.message"),
```

Comment:
Sorry I'm just rebasing on this and noticed - it feels a little duplicative to call out both `index` and `start` in this key. Is there more than 1 header / message on this page?

Reply:
Ah, based on the PO it looks like these are for the item in the media list yes? I'm just trying to get the most clear keys here so we all know what's going on.

Reply:
No worries, I appreciate this discussion for sure! The header/message on lines 111 and 112 go with the first media item, which is dynamic per-verifier. The other media item on the page is the bank card one, which is always the same. I've labeled them with their current msgids in the screenshot below:

![image](https://user-images.githubusercontent.com/25497886/193627446-1db985fc-4349-4e12-90ec-37542101ba91.png)

Reply:
Wait... isn't that a screenshot of the Eligibility Start page? This ticket is for the Enrollment start/index right?

Reply:
What if the keys matched, like

```
enrollment_start_heading=_("enrollment.pages.index.mst_login.start.heading"),
enrollment_start_details=_("enrollment.pages.index.mst_login.start.details"
```

```
        confirmed_benefit_item = dict(
            icon=viewmodels.Icon("happybus", pgettext("image alt text", "core.icons.happybus")),
            heading=_(verifier.enrollment_start_heading),
            details=_(verifier.enrollment_start_details),
        )
```

Reply:
I see why @angela-tran you had the `confirmed` string in the key earlier. If we made it `eligibility_confirmed` would that work here?

Reply:
You're right @thekaveman , my bad. Here's the Enrollment start/index

![image](https://user-images.githubusercontent.com/25497886/193628987-19055a03-993e-4553-9ab5-f5c73cd5dd61.png)

Reply:
I've renamed the properties and msgids that this PR added. I know there will be some other renaming in #994 and #376, so I tried to keep my renaming here minimal while still correct. I ran `./bin/makemessages.sh` so I think it should all be correct. Let me know if you see any other mistakes.